src/components/app/ProjectPreview.svelte

```diff
@@ -56,6 +99,8 @@
         representativeBackground,
         representativeFace,
         representativeText,
+        isCustomCharacter,
```

Comment:
Why is this field necessary? It seems like it's redundant with characterName, which is null if none was found. I recommend remove it, and just using the name.